src/DataService/DataServiceDiscovery.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Lullabot\Mpx\DataService;
+
+use Doctrine\Common\Annotations\Reader;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * Class to discover Data Service implementations.
+ */
+class DataServiceDiscovery
```

Comment:
My guess is that a Drupal or other framework implementation would replace this class, but we still need it for our own testing, and it's useful as a guide.